pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -458,48 +462,52 @@ func (pe *StandardSecurityPolicyEnforcer) EnforceCreateContainerPolicy(
 	user IDName,
 	groups []IDName,
 	umask string,
-) (allowedEnvs EnvList, stdioAccessAllowed bool, err error) {
+	caps *oci.LinuxCapabilities,
+) (allowedEnvs EnvList,
+	allowedCapabilities *oci.LinuxCapabilities,
+	stdioAccessAllowed bool,
+	err error) {
```

Comment:
it may be less disruptive to create a `CreateContainerResult` struct that we can add more fields to rather than keep expanding the return items (if more are to be added)

Reply:
Also I think this "disruptive" is a good thing. It allows the compiler to help us not overlook places where an additional return value needs to be handled. Putting it in a return object means we no longer have the compiler able to help us not overlook possible points that need to be updated (be it in the uvm, the simulator, tests etc). This helps us avoid a very hard to find and insidious form of bug.